test/regression/expected/views.out

```diff
@@ -27,9 +27,13 @@ create table "s.t" as select 42;
 create view vw1 as select * from s.t;
 create view vw2 as select * from "s.t";
 select * from vw1, vw2;
+WARNING:  (DuckDB) Binder Error: Referenced table "t" not found!
+Candidate tables: "s.t"
+LINE 3:    FROM ( SELECT t."?column?"
+                         ^
  ?column? | ?column? 
 ----------+----------
-       21 |       21
+       21 |       42
```

Comment:
The previous DuckDB pushdown was actually returning the wrong results here. So the DuckDB failure that this introduces is better than the alternative imho. Because now it warns and automatically falls back to Postgres, instead of returning a wrong result.

Reply:
Yea I noticed that as well, the postgres catalog extension solved the same problem